zerver/lib/test_helpers.py

```diff
@@ -540,7 +540,7 @@ def find_pattern(pattern: Any, prefixes: list[str]) -> None:
             "docs/",
             "docs/(?P<path>.+)",
             "casper/(?P<path>.+)",
-            "static/(?P<path>.+)",
+            "static/(?P<path>.*)",
```

Comment:
An odd side effect of removing the `zerver.tests.test_urls` → `zproject.urls` import is that when `zerver.lib.test_helpers` eventually imports `zproject.urls`, that happens under `settings.DEBUG = True` rather than `settings.DEBUG = False`, which results in this slight change to the regex for `static/`.